amp-script-tutorial/README.md

```diff
@@ -0,0 +1,29 @@
+# amp-script-tutorial
+
+## About
+
+This repository contains the code you'll use to do the [&lt;amp-script&gt; tutorial](https://amp.dev/documentation/guides-and-tutorials/develop/custom-javascript-tutorial/)!
+
+This tutorial presents a form containing a password input. We want your password to follow certain rules, and so the user can't submit the form until those rules have been followed. In the tutorial, you begin with HTML that enforces the rules using &lt;amp-form&gt;'s `pattern` attribute. You then transform that into a more user-friendly experience which uses &lt;amp-script&gt;.
+
+This repo contains two subdirectories:
+* `starter_code`: the code you start with
+* `finished_code`: what you'll end up with after you complete the tutorial
+
+Note that this code doesn't include a server, so if you actually submit the form, you'll get an error in the Console. This exercise is left for the reader 😎
```

Comment:
Suggestion to change `This exercise is left for the reader 😎`  to something like `Use your preferred method or one of the setup suggestions below to spin up a sever.`

Reply:
I'd have to also instruct the reader to not just create a server, as in the original tutorial, but also to include a service that handles submissions (which the original tutorial doesn't do). I'd rather keep this short and sweet unless you think I should provide this explicit guidance.

Reply:
A tutorial should provide every single step the follower needs to succeed - as in I should never have to follow a different tutorial to successfully reach the end project. So I'd say provide explicit guidance in this case.

Reply:
ok!